src/frontend/src/components/Posts.svelte

```diff
@@ -17,39 +18,77 @@
+        <path stroke-linecap="round" stroke-linejoin="round" d="M15 12a3 3 0 11-6 0 3 3 0 016 0z"></path>
+      </svg>
+    </button>
+  </Navigate>
+</div>
+<div class="max-w-xl mx-auto mt-8">
+  <div class=" flex items-center">
+    <!-- Avatar Picture -->
+    <div class="avatar">
+      <div class="w-32 rounded mr-4">
+        <img src="{profile.avatar_url}" alt="avatar"/>
+      </div>
     </div>
-    <div>
-      {profile.about}
+
+    <!-- Username & Bio Container -->
+    <div class="ml-4">
+        <h1 class="text-xl font-bold">@{profile.username}</h1>
+        <p class="text-sm">{profile.about}</p>
     </div>
   </div>
 </div>
 
-<div class="mt-12">
-  <div>Create post</div>
-  <input bind:value={ message } placeholder="Input message" class="input input-bordered"/>
-  <button
-    disabled="{ !message }"
-    class="btn btn-primary"
-    on:click={async () => await create_post()}
-  >
-    Create
-  </button>
+<div class="max-w-xl mx-auto mt-12">
+  <div class="divider"></div>
+  <div class="form-control">
+    <label class="label">
+      <span class="label-text text-base">Create a new post</span>
+    </label>
+    <textarea bind:value={message} class="textarea textarea-bordered" maxlength="200" placeholder="Write something..."></textarea>
+  </div>
+  <div class="text-right">
+    <button
+      disabled="{ !message }"
+      class="btn btn-primary mt-2 text-right"
+      on:click={async () => await create_post()}
+    >
+      Create
+    </button>
+  </div>
+  <div class="divider"></div>
+  <div class="mt-12">
+    <!-- <div class="text-base">Your posts</div> -->
+    {#each $nostr_events as event}
+      <div class="post mb-4 p-2 rounded">
+        <div class=" flex items-start">
+          <!-- Avatar Picture -->
+          <div class="avatar">
+            <div class="w-16 rounded mr-4">
+              <img src="{profile.avatar_url}" alt="avatar"/>
+            </div>
+          </div>
+      
+          <!-- Username & Bio Container -->
+          <div class="ml-2">
+              <div>{event.created_at}</div>
```

Comment:
tento formát dátumu by chcelo upraviť len na klasický MM:HH:SS, ideálne už tam v nostr servise pri sub callbacku, keď než sa to hodí do nostr storu.